### PR TITLE
[ready to merge] 4.7.17 release notes

### PR DESCRIPTION
**Pull request by agh1**
This is a first run of the release notes.   The issues need review, plus formatting changes.

### CODE REVIEW

File: release-notes/4.7.17.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+- **Release notes 4.7.16 (rest of 'em) ([9803](https://github.com/civicrm/civicrm-core/pull/9803))**
+
+- **remove markdown files in /docs ([9771](https://github.com/civicrm/civicrm-core/pull/9771))**
+
+- **civicrm_webtest.install - Restore access to edit message templates and tags ([9785](https://github.com/civicrm/civicrm-core/pull/9785))**
+
+- **Update Test generated examples ([9798](https://github.com/civicrm/civicrm-core/pull/9798))**
+
+- **Release notes 4.7.16 [wip] ([9781](https://github.com/civicrm/civicrm-core/pull/9781))**
```
Comment (seamuslee001) [inline release-notes/4.7.17.md]: @agh1 may not need this one

Comment (agh1) [inline release-notes/4.7.17.md]: :smirk: yep. This is raw from the script I use to scrape the commits since 4.7.16.

**Comment by seamuslee001** [comment]
Jenkins test this please

**Comment by agh1** [comment]
@seamuslee001 poor Jenkins--running tests for a PR exclusively made up of markdown files!

**Comment by xurizaemon** [comment]
Will it help if we test some markdown too?

**Comment by seamuslee001** [comment]
@agh1 Andrew just want to make sure you got these captured 

https://issues.civicrm.org/jira/browse/CRM-20004 https://issues.civicrm.org/jira/browse/CRM-20201 as they were regressions in the RC found and fixed up

**Comment by agh1** [comment]
The notes are all set to merge @totten @colemanw. 
@guanhuan @mi-kon you're welcome to review.
@seamuslee001 thanks for highlighting those late merges--they're included.

**Pull request merged**